front/src/modules/timesStops/TimesStops.tsx

```diff
@@ -112,10 +137,6 @@ const TimesStops = ({ pathProperties, pathSteps = [] }: TimesStopsProps) => {
           })
         ),
         title: `${t('stopTime')}`,
-
-        // We should not be able to edit the stopping time of the origin and destination
-        disabled: ({ rowIndex }) =>
-          rowIndex === 0 || rowIndex === pathProperties.allVias?.length - 1,
```

Comment:
Why do we want to edit the stop time of the origin or destination ?

Reply:
That's a request from @bgiuliana. Apparently the users said they can sometimes need to add a stop duration to these steps.

Reply:
Ok but we should check with @flomonster because I think it will break on their side :/

Reply:
We did. For the origin, it's just if we put an `arrival` that it will break. And for the destination, only margin is forbidden.